Fixed UI capitalization and style errors.

Ismith507 opened the pull request:
## Summary of the Pull Request
This pull request addresses the issue opened by @PhMajerus concerning the UI and style errors in Windows Terminal

## Detailed Description of the Pull Request / Additional comments
PR is pretty straightforward. Just edited src/cascadia/TerminalApp/Resources/en-US/Resources.resw to reflect the changes desired in the issue.

This is my first contribute to a repository that is not my own. So, if I am missing any requirements in my PR or if anything needs to be changed, please let me know!

Closes #16846

Ismith507 commented:
@microsoft-github-policy-service agree

carlos-zamora commented on CppProperties.json:
```diff
@@ -0,0 +1,21 @@
+{
```
undo checking in this file, please

carlos-zamora commented on src/cascadia/TerminalApp/Resources/en-US/Resources.resw:
```diff
@@ -194,28 +194,28 @@
   </data>
   <data name="TabCloseOther" xml:space="preserve">
-    <value>Close Other Tabs</value>
+    <value>Close other Tabs</value>
```
```suggestion
    <value>Close other tabs</value>
```
Shouldn't "Tabs" be changed to "tabs" too? The open issue said that we should use "Sentence-style capitalization".

carlos-zamora commented on src/cascadia/TerminalApp/Resources/en-US/Resources.resw:
```diff
@@ -472,27 +472,27 @@
   </data>
   <data name="AboutDialog_ThirdPartyNoticesLink.Content" xml:space="preserve">
-    <value>Third-Party Notices</value>
+    <value>Third-Party notices</value>
```
```suggestion
    <value>Third-party notices</value>
```
Sentence style capitalization means the "p" should be lowercase

carlos-zamora commented on src/cascadia/TerminalApp/Resources/en-US/Resources.resw:
```diff
@@ -618,31 +618,31 @@
   <data name="MediumSeaGreenColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Medium Sea Green</value>
+    <value>Medium sea green</value>
   </data>
   <data name="DarkOrangeColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Dark Orange</value>
+    <value>Dark orange</value>
   </data>
   <data name="MediumVioletRedColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Medium Violet Red</value>
+    <value>Medium violet red</value>
   </data>
   <data name="DodgerBlueColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Dodger Blue</value>
+    <value>Dodger blue</value>
   </data>
   <data name="LimeGreenColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Lime Green</value>
+    <value>Lime green</value>
```
I think all of these should stay as-is, actually. They're proper nouns of the color, so guidelines state we should capitalize all of it.

carlos-zamora commented on src/cascadia/TerminalApp/Resources/en-US/Resources.resw:
```diff
@@ -618,31 +618,31 @@
   <data name="BlueVioletColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Blue Violet</value>
+    <value>Blue violet</value>
   </data>
   <data name="SlateBlueColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Slate Blue</value>
+    <value>Slate blue</value>
```
Same goes for these, should probably stay capitalized since they're proper nouns for colors.

carlos-zamora commented on src/cascadia/TerminalApp/Resources/en-US/Resources.resw:
```diff
@@ -657,10 +657,10 @@
   <data name="SkyBlueColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Sky Blue</value>
+    <value>Sky blue</value>
   </data>
   <data name="DarkGrayColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Dark Gray</value>
+    <value>Dark gray</value>
```
Same goes for these, should probably stay capitalized since they're proper nouns for colors.

carlos-zamora reviewed (changes requested):
Thanks for contributing! Sorry it's taken a while for us to review this. 

You're talking about #16846, right? I went ahead and edited your PR description, but normally, update the "Closes #xxx" text in the template to be "Closes #16846". If you want to be extra nice, remove all the extra stuff in the PR description too, but not strictly necessary. We just like linking up the relevant issues so we know what the PR is trying to address 😊.

Looking through this PR, I think updating the project files is out of scope for what you're trying to do. Please undo changes to the following files (all of them except the resw file):
- OpenConsole.sln
- src/cascadia/WindowsTerminal_UIATests/WindowsTerminal.UIA.Tests.csproj
- src/cascadia/WindowsTerminal_UIATests/app.config
- src/cascadia/WpfTerminalControl/WpfTerminalControl.csproj
- src/cascadia/WpfTerminalTestNetCore/WpfTerminalTestNetCore.csproj
- src/host/ft_uia/Host.Tests.UIA.csproj
- src/host/ft_uia/app.config
- src/tools/TerminalStress/TerminalStress.csproj
- src/tools/vtapp/App.config
- src/tools/vtapp/VTApp.csproj

Other than that, I think we're good! I'll go ahead and approve once the changes go in. Thanks!

carlos-zamora commented:
Oh! And normally you'd have to worry about the spell-checker. But I think that it's just flagging stuff in the project files that need to be untracked, so you shouldn't have to worry about it.

Ismith507 commented:
Hello!

I created a new PR (#17037) that is ready for your review. I have corrected the mistakes in this PR and implemented some of the changes you suggested. I'm not sure how so many files got changed when I was working on my edits, it must have been when I was trying to build and run the project on my computer. Anyway, this new PR should limit its changes to the Resources.resw file. 

Thank you for your input!
